gatsby-node.js

```diff
@@ -115,13 +117,130 @@ exports.sourceNodes = async ({ actions }) => {
   return
 }
 
-// exports.onCreateNode = ({ actions, node }) => {
-//   const { type } = node.internal
-//   if (type !== `Member`) return
-//   const { bio } = node
-//   const bioAsHtml = markdownProcessor.process(bio)
-//   node.bio = bioAsHtml
-// }
+exports.onCreateNode = ({ node, actions }) => {
+  const { type } = node.internal
+  if (type === 'GhostAuthor') {
+    if (node.slug !== 'data-schema-author') {
+      actions.createNodeField({
+        node,
+        name: `member___NODE`,
+        value: `Labs__Member__${node.slug}`,
+      })
+    }
+  }
+}
+
+const createTagPages = (tags, createPage) => {
+  const tagsTemplate = path.resolve(`./src/templates/tag.tsx`)
+
+  tags.forEach(({ node }) => {
+    const totalPosts = node.postCount !== null ? node.postCount : 0
+    const numberOfPages = Math.ceil(totalPosts / postsPerPage)
+
+    // This part here defines, that our tag pages will use
+    // a `/tag/:slug/` permalink.
+    node.url = `/tag/${node.slug}/`
+
+    Array.from({ length: numberOfPages }).forEach((_, i) => {
+      const currentPage = i + 1
+      const prevPageNumber = currentPage <= 1 ? null : currentPage - 1
+      const nextPageNumber =
+        currentPage + 1 > numberOfPages ? null : currentPage + 1
+      const previousPagePath = prevPageNumber
+        ? prevPageNumber === 1
+          ? node.url
+          : `${node.url}page/${prevPageNumber}/`
+        : null
+      const nextPagePath = nextPageNumber
+        ? `${node.url}page/${nextPageNumber}/`
+        : null
```

Comment:
Might be good to put this in some helper functions bc the ternaries all next to each other are kinda hard to parse

Reply:
A lot of this code was copy/pasted from the gatspy-ghost starter repo... this code for doing pagination was just completely unnecessary, since they also imported a library for gatsby pagination. It's been removed/replaced with the library code.